### PR TITLE
Adjustments to include EVerest as upstream project

### PR DESCRIPTION
**Pull request by dorezyuk**
## Describe your changes

Adjusts the bazel paths to allow users to include EVerest-core as an upstream project

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements

### CODE REVIEW

**Review by SirVer — APPROVED** [review]

**Comment by hikinggrass** [comment]
@dorezyuk your commit is missing a sign-off before we can merge it

**Review by corneliusclaussen — APPROVED** [review]

**Comment by corneliusclaussen** [comment]
@golovasteek your review is missing here

File: interfaces/powermeter.yaml
Comment (corneliusclaussen) [inline interfaces/powermeter.yaml:29]: In the mean time this was added in core as well but with a different name: public_key_ocmf

**Review by corneliusclaussen — CHANGES_REQUESTED** [review]

**Review by corneliusclaussen — APPROVED** [review]

**Pull request merged**
